frontend-nextjs/__tests__/page.test.tsx

```
@@ -1,8 +1,36 @@
+
+// FileUploadコンポーネントのモック
+vi.mock('@/app/components/FileUpload', () => ({
+  default: () => <div data-testid="file-upload">File Upload Component</div>,
+}));
+
+// next/imageのモック
+vi.mock('next/image', () => ({
+  default: (props: any) => <img {...props} />,
+}));
+
+describe('UploadPage', () => {
```

Biome

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.

(lint/a11y/useAltText)